src/shared/common/forms/mapped-display.jsx

```diff
@@ -36,7 +36,7 @@ class MappedDisplay extends Component {
             return (renderCheckbox(label, this.props.showLabel, choices[0].label));
         } else {
             return (
-                <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value}</span>
+                <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value.split(',').join(', ')}</span>
```

Comment:
You could use replaceAll here instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

My only thing is I believe the split and join is O(n^2) performance whereas I'd presume this is O(n).